uber/site_sections/summary.py

```diff
@@ -134,19 +134,35 @@ def shirt_counts(self, session):
         status = lambda got_merch: 'picked_up' if got_merch else 'outstanding'
         for attendee in session.query(Attendee).all():
             if attendee.gets_free_shirt:
-                counts['free'][label(attendee.shirt_label)][status(attendee.got_merch)] += 1
```

Comment:
That's true, and I was just thinking about that while reviewing this.

The original issue specifically asked for it to be split up like this, but it's probably better if we make sure to include everything.